openshift/backend.dc.yaml

```diff
@@ -183,16 +183,16 @@ objects:
               secretKeyRef:
                 key: password
                 name: getok-sc-getokprod-secret
-          - name: SC_MSSC_USERNAME
+          - name: CHES_SC_USERNAME
             valueFrom:
               secretKeyRef:
                 key: username
-                name: getok-sc-mssc-secret
-          - name: SC_MSSC_PASSWORD
+                name: getok-sc-ches-secret
+          - name: CHES_SC_PASSWORD
```

Comment:
Just caught this but we should be keeping the variable naming order `SC_CHES_PASSWORD` to stay aligned with how we're structuring the config.json object structure. Either way it does get the job done, but the values should try to keep some conformity as it'll help other developers who aren't as intimately familiar with the codebase be able to pick things up at a glance.

Reply:
Refactoring some config setup as part of next PR likely.